Gemfile

```diff
@@ -1,11 +1,13 @@
 source 'https://rubygems.org'
 
-gem 'berkshelf',  '~> 3.2'
-gem 'chefspec',   '~> 4.2'
-gem 'foodcritic', '~> 4.0'
-gem 'rubocop',    '~> 0.30.1'
+gem 'berkshelf', '~> 4.3.5'
```

Comment:
Do you mean not to pin by versions at all? Sorry a bit late here and perhaps I did not get clearly what you meant :-)